client/app/modules/tests-sessions/test-session-card/test-session-card.scss

```diff
@@ -0,0 +1,322 @@
+@import "../../../../styles/global/var";
+@import "../../../../styles/ui/components/animation";
+
+.test-session-card {
+    &__clickable {
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        right: 0;
+        left: 0;
+        z-index: 1;
+        &:hover {
+            background-color: rgba(0,0,0,.05);
+        }
+        &._highlighting {
+            animation: change-color 3s ease-in-out 1;
+        }
+    }
+    &__wrapper {
+        position: relative;
+        background-color: #fff;
+        border-left: 4px solid #777;
+        min-height: 55px;
+        display: flex;
+
+        &-left,
+        &-right {
+            display: flex;
+        }
+        &-left {
+            width: 40%;
+            flex-grow: 1;
+            min-width: 400px;
+        }
+        &-right {
+            width: 60%;
+            position: relative;
+        }
+        &.COMPLETED {
+            border-left-color: $green-new;
+        }
+
+        &.FAILED {
+            border-left-color: $red-new;
+        }
+
+        &.IN_PROGRESS {
+            border-left-color: $blue-new;
+        }
+
+        &.TIMEOUT {
+            border-left-color: rgba($gray-light-new, .5);
+        }
+    }
+    &__cell {
+        display: flex;
+        align-items: center;
+        color: rgba(0, 0, 0, 0.87);
+        font-size: 13px;
+        border-top: 1px rgba(0, 0, 0, 0.12) solid;
+        flex-shrink: 0;
+        padding: 5px;
+
+        &._info {
+            flex-grow: 1;
+            flex-direction: column;
+            align-items: flex-start;
+            justify-content: center;
+            flex-shrink: 1;
+            min-width: 200px;
+            padding-left: 11px;
+        }
+        &._env {
+            justify-content: center;
+            flex-grow: 1;
+            width: 11%;
+            min-width: 100px;
+            & .label {
+                font-weight: bold !important;
```

Comment:
Ivan taught us to use numeric values for font-weight

Reply:
fixed